nupic/research/frameworks/vernon/experiments/cl_experiment.py

```diff
@@ -138,7 +138,8 @@ def create_validation_sampler(cls, config, dataset):
     def compute_task_indices(cls, config, dataset):
         # Assume dataloaders are already created
         class_indices = defaultdict(list)
-        for idx, (_, target) in enumerate(dataset):
+        for idx, batch_item in enumerate(dataset):
```

Comment:
See comment on preserving the dataloader signature. It is a small change and won't really modify things a lot here, just how you unpack.